include/dsn/utility/error_code.h

```diff
@@ -124,4 +124,5 @@ DEFINE_ERR_CODE(ERR_UNAUTHENTICATED)
 DEFINE_ERR_CODE(ERR_KRB5_INTERNAL)
 
 DEFINE_ERR_CODE(ERR_SASL_INTERNAL)
+DEFINE_ERR_CODE(ERR_NOT_COMPLEMENTED)
```

Comment:
What ERR_NOT_COMPLEMENTED suggests? I didn't see the meaning anywhere.

Reply:
`ERR_NOT_COMPLEMENTED` means the security process is not complemented, that we should process the next step by `sasl->step`.
First I used the `ERR_NOT_IMPLEMENTED`, but the meaning of this error code is not accure  accurate  for this case

Reply:
How about `ERR_SASL_INCOMPLETE` ?

Reply:
good idea.